src/inventory/disks.go

```diff
@@ -124,6 +124,26 @@ func (d *disks) getBootable(path string) bool {
 	return strings.Contains(stdout, "DOS/MBR boot sector")
 }
 
+func (d *disks) hasUUID(path string) bool {
+	if path == "" {
+		return false
+	}
+
+	// Openshift requires VSphere (and maybe other platforms) VMs to have a UUID for disks
+	// Here we add this information to the inventory for the assisted-service verification.
+	// Getting device page 83 which contains the device WWID(disk.UUID)
+	// please see: https://access.redhat.com/solutions/93943
+	stdout, stderr, exitCode := d.dependencies.Execute("sg_inq", "-p", "0x83", path)
+
+	if exitCode != 0 {
+		logrus.Warnf("Could not get UUID information for path %s: %s", path, stderr)
```

Comment:
I'm afraid users will see this and get scared. Can you rephrase it to something less alerting, maybe something like:

```suggestion
		logrus.Infof("hasUUID is false for path %s: %s", path, stderr)
```

Reply:
hasUUID is false isn't clear enough. 
Updated to info

Reply:
>hasUUID is false isn't clear enough.

I know, it's intentionally vague to make naive users ignore it.

Sadly our users don't care about the severity of the messages. They look for error-looking messages everywhere (even inside the inventory itself, and constantly post them on our Slack channel and open BZs about them.

If the user has even 1 disk that triggers this , then they're gonna see this messages show up in the agent logs every 60 seconds (everytime there's an inventory step), it's gonna look scary and they're gonna waste their time on it, report it as a bug, or ask about it on Slack, wasting even more people's time.

I'm not just speculating, this is how our users behave, they look for error-looking strings *[everywhere](https://issues.redhat.com/browse/MGMT-7388)*.

Reply:
thanks for clarifying